nameko/testing/services.py

```diff
@@ -261,43 +294,45 @@ def cm_to_inches(self, cms):
                 return self.maths_rpc.divide(cms, 2.54)
 
         container = ServiceContainer(ConversionService, config)
-        maths_rpc = replace_dependencies(container, "maths_rpc")
+        mock_maths_rpc = replace_dependencies(container, "maths_rpc")
 
         container.start()
 
         with ServiceRpcProxy('conversionservice', config) as proxy:
             proxy.cm_to_inches(100)
 
         # assert that the dependency was called as expected
-        maths_rpc.divide.assert_called_once_with(100, 2.54)
+        mock_maths_rpc.divide.assert_called_once_with(100, 2.54)
 
-    """
-    if container.started:
-        raise RuntimeError('You must replace dependencies before the '
-                           'container is started.')
+    **Kwargs Usage**
 
-    dependency_names = {dep.attr_name for dep in container.dependencies}
+    ::
 
-    missing = set(dependencies) - dependency_names
-    if missing:
-        raise ExtensionNotFound("Dependency(s) '{}' not found on {}.".format(
-            missing, container))
+        mock_maths_rpc = MagicMock()
```

Comment:
This isn't the most compelling example because it doesn't achieve anything different from the other API. How about the following?

```python
stub_maths_rpc = RpcProxy('stub-maths')
```

You'd need to drop the assertion about the mock too. Maybe just state what you would do, rather than demonstrate it...